packages/origin-ui-irec-core/src/components/detail-view/DeviceDetailView.tsx

```diff
@@ -0,0 +1,324 @@
+                        deviceExternalId.id === props.externalId.id &&
+                        deviceExternalId.type === props.externalId.type
+                )
+            );
+            setSelectedDevice(device);
+        }
+    }, [props.id, props.externalId]);
+
+    // useEffect(() => {
+    //     const getDeviceOrg = async () => {
+    //         const organization = await backendClient.organizationClient.getPublic(selectedDevice.ownerId);
+    //         setSelectedDevOrg(organization);
+    //     }
+    // }, [selectedDevice])
+
+    if (!configuration || !organizationClient || !selectedDevice) {
+        return <Skeleton variant="rect" height={200} />;
+    }
+
+    let tooltip = '';
+
+    const selectedDeviceType = selectedDevice.deviceType;
+    let image = solar;
+
+    if (selectedDeviceType.startsWith('Wind')) {
+        image = wind;
+    } else if (selectedDeviceType.startsWith('Hydro-electric Head')) {
+        image = hydro;
+    } else if (selectedDeviceType.startsWith('Thermal')) {
+        image = iconThermal;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Adam Terpening' });
+    } else if (selectedDeviceType.startsWith('Solid')) {
+        image = iconSolid;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'ahmad' });
+    } else if (selectedDeviceType.startsWith('Liquid')) {
+        image = iconLiquid;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'BomSymbols' });
+    } else if (selectedDeviceType.startsWith('Gaseous')) {
+        image = iconGaseous;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Deadtype' });
+    } else if (selectedDeviceType.startsWith('Marine')) {
+        image = iconMarine;
+        tooltip = t('general.feedback.createdByNoun', { fullName: 'Vectors Point' });
+    }
+
+    const data = [
+        [
+            {
+                label: t('device.properties.facilityName'),
+                data: selectedDevice?.name
+            },
+            {
+                label: t('device.properties.deviceOwner'),
+                data: selectedDevice?.registrantOrganization
+            },
+            {
+                label: t('device.properties.complianceRegistry'),
+                data: 'what is instead?' // selectedDevice.complianceRegistry
+            },
+            {
+                label: t('device.properties.otherGreenAttributes'),
+                data: 'what is instead?' // selectedDevice.otherGreenAttributes
+            }
+        ],
+        [
+            {
+                label: t('device.properties.deviceType'),
+                data: selectedDevice?.deviceType,
+                image,
+                rowspan: 2
+            },
+            {
+                label: t('device.properties.meterReadCertified'),
+                data: 'what is instead?', // EnergyFormatter.format(selectedDevice.meterStats?.certified ?? 0),
+                tip: EnergyFormatter.displayUnit
+            },
```

Comment:
This to be read from the `issuer-client` https://github.com/energywebfoundation/origin/blob/ec5aa08791af0abfe3b1ee059657518dff68c05e/packages/issuer-api/src/pods/certificate/certificate.controller.ts#L93